Identity/Identity.API/Helpers/JwtGenerator.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Text;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using Microsoft.IdentityModel.Tokens;
+using Identity.DataAccess.Models.Entities;
+using Identity.API.Options;
+using Microsoft.Extensions.Options;
+
+namespace Identity.API.Helpers;
+
+public class JwtGenerator(IOptions<JwtOptions> jwtOptions)
+{
+    private readonly JwtOptions _jwtOptions = jwtOptions.Value;
+
+    public string GenerateJwtToken(UserEntity user)
+    {
+        var value = DateTime.Now.AddHours(2);
+        var bytes = Encoding.UTF8.GetBytes(_jwtOptions.Key);
+        var signingCredentials = new SigningCredentials(new SymmetricSecurityKey(bytes), SecurityAlgorithms.HmacSha256);
+        
+        var claims = new Dictionary<string, object>
+        {
+            { ClaimTypes.NameIdentifier, user.Id },
+            { ClaimTypes.Name, user.Username },
+            { ClaimTypes.Role, user.Roles },
+            { ClaimTypes.Email, user.Email },
```

Comment:
There's some things here that [shouldn't be in claims](https://learn.microsoft.com/en-us/entra/identity-platform/migrate-off-email-claim-authorization) (Username and Email specifically).